src/components/Button/Button.module.css

```diff
@@ -139,19 +138,27 @@
  */
 .button--sm {
   @mixin eds-theme-typography-button-sm;
-  padding-top: var(--eds-size-half);
-  padding-right: var(--eds-size-1);
-  padding-bottom: var(--eds-size-half);
-  padding-left: var(--eds-size-1);
+  height: var(--eds-size-3);
```

Comment:
We're specifying height instead of top and bottom padding because of our icons, which are enlarged when placed in buttons so they match the text even though they would cause the button height to expand if not contained in this way.